scss-test.scss

```diff
@@ -0,0 +1,16 @@
+.unsorted {
+  color: #FFFFFF;
+  background: red;
+  .inception-1 {
+    color: #DDD;
+    .inception-2 {
+       color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 6 spaces, but was indented 7 spaces